src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -0,0 +1,44 @@
+import { incidentDate, secondaryIncidentDate } from '../../pages';
+
+import { needsToEnter781, needsToEnter781a } from '../../utils';
+
+export function formConfig781(iterations) {
+  let configObj = {};
+  for (let index = 0; index < iterations; index++) {
+    configObj = {
+      ...configObj,
+      // 781 PAGE CONFIGS GO HERE
+      ...{
+        [`incidentDate${index}`]: {
+          title: 'PTSD Incident Date',
+          path: `disabilities/ptsd-incident-date-${index}`,
+          depends: needsToEnter781,
+          uiSchema: incidentDate.uiSchema(index),
+          schema: incidentDate.schema(index),
+        },
+      },
+    };
+  }
+  return configObj;
+}
+
+export function formConfig781a(iterations) {
+  let configObj = {};
+  for (let index = 0; index < iterations; index++) {
+    configObj = {
+      ...configObj,
+      // 781a PAGE CONFIGS GO HERE
+      ...{
+        [`secondaryIncidentDate${index}`]: {
+          title: 'PTSD Incident Date',
+          path: `disabilities/ptsd-secondary-incident-date-${index}`,
+          // The Depends will need to be refactored to account for the page index/incident Number
```

Comment:
Yeah, for both 781 & 781a I'd think that you only want to show the right number of pages, even if you create a config with all 3 of the iteratee pages ahead of time.

Reply:
We'll have our "Is there another incident you would like to tell us about" `yesNo` question at the end of each incident set. We can conditionally check each page after that to the previous incident sets yesNo question.